.travis.yml

```diff
@@ -1,99 +1,84 @@
 language: php
-
-sudo: required
-dist: trusty
+os: linux
+dist: xenial
 
 cache:
   directories:
     - $HOME/.composer/cache
     - $HOME/.npm
 
 addons:
-  firefox: "47.0.1"
-  postgresql: "9.4"
+  postgresql: "9.5"
+
+services:
+  - mysql
+  - docker
 
 php:
-  - 7.0
-  - 7.1
+  - 7.2
   - 7.3
-
-matrix:
- exclude:
-   - php: 7.0
-     env: DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE
-   - php: 7.0
-     env: DB=mysqli MOODLE_BRANCH=MOODLE_37_STABLE
-   - php: 7.0
-     env: DB=pgsql MOODLE_BRANCH=MOODLE_38_STABLE
-   - php: 7.0
-     env: DB=mysqli MOODLE_BRANCH=MOODLE_38_STABLE
-   - php: 7.0
-     env: DB=pgsql MOODLE_BRANCH=master
-   - php: 7.0
-     env: DB=mysqli MOODLE_BRANCH=master
-   - php: 7.3
-     env: DB=pgsql MOODLE_BRANCH=MOODLE_35_STABLE
-   - php: 7.3
-     env: DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE
- fast_finish: true
-
+  - 7.4
 
 env:
- matrix:
-  - DB=pgsql MOODLE_BRANCH=MOODLE_35_STABLE
-  - DB=pgsql MOODLE_BRANCH=MOODLE_36_STABLE
-  - DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE
-  - DB=pgsql MOODLE_BRANCH=MOODLE_38_STABLE
-  - DB=pgsql MOODLE_BRANCH=master
-  - DB=mysqli MOODLE_BRANCH=MOODLE_35_STABLE
-  - DB=mysqli MOODLE_BRANCH=MOODLE_36_STABLE
-  - DB=mysqli MOODLE_BRANCH=MOODLE_37_STABLE
-  - DB=mysqli MOODLE_BRANCH=MOODLE_38_STABLE
-  - DB=mysqli MOODLE_BRANCH=master
+  jobs:
+    - DB=pgsql MOODLE_BRANCH=MOODLE_35_STABLE
+    - DB=pgsql MOODLE_BRANCH=MOODLE_37_STABLE
```

Comment:
_36_ is missing; is that intentional?

Reply:
Yes, the image seems a bit weird, but I think support for 3.6 ends this month: https://docs.moodle.org/dev/Releases#Version_support

Reply:
also I noticed that the new node version seems to conflict with Moodle 3.6, so it is probably better this way :roll_eyes: